packages/documentation/docs/controls/charts/overview.md

```diff
@@ -25,6 +28,14 @@ import { registerTheme } from '@siemens/ix-echarts';
 registerTheme(echarts);
 ```
 
+<div className="siemens-brand-section">
```

Comment:
What should do the `siemens-brand-section`?

Reply:
removed